FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -1074,11 +1069,30 @@ extension ComposeViewController {
 
 // MARK: - Action Handling
 extension ComposeViewController {
+    func encryptAndSend(message: SendableMsg, threadId: String?) async throws {
+        try await composeMessageService.encryptAndSend(
+            message: message,
+            threadId: threadId
+        )
+    }
+
+    func searchContacts(query: String) async throws -> [String] {
+        return try await cloudContactProvider.searchContacts(query: query)
+    }
+
+    func findLocalContact(with email: String) async throws -> RecipientWithSortedPubKeys? {
+        return try await localContactsProvider.searchRecipient(with: email)
+    }
+
+    func fetchContact(with email: String) async throws -> RecipientWithSortedPubKeys {
+        return try await pubLookup.fetchRemoteUpdateLocal(with: email)
+    }
```

Comment:
These methods are no longer needed at all - the consumers in this class can be rewritten to use the direct methods like `await pubLookup.fetchRemoteUpdateLocal(with: email)` instead of `fetchContact(with email: String)`

Reply:
Same probably applies to most of the rest as well